Dockerfile.builder

```diff
@@ -15,6 +16,14 @@ RUN yum install -y \
       which \
     && yum clean all
 
+RUN wget https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz \
+    && tar -xvf go$GO_VERSION.linux-amd64.tar.gz \
+    && mv go /usr/local \
```

Comment:
@baijum I'd say, that if /usr/local/go exists. We're doing something wrong. No need to force it here, I guess.